.github/workflows/nightfalldlp.yml

```diff
@@ -0,0 +1,22 @@
+name: nightfalldlp
```

Comment:
Is the point of this workflow to just test that the action is working correctly?

Reply:
yep!